cordova-lib/package.json

```diff
@@ -40,7 +40,7 @@
         "underscore": "1.7.0",
         "unorm": "1.3.3",
         "valid-identifier": "0.0.1",
-        "xcode": "0.8.0"
+        "xcode": "0.8.6"
```

Comment:
Would you guys prefer I point that by git fork instead just to get the build/tests passing? I'd think pointing at a non-released version would also make it so this is not mergeable, so not sure what the best state is to leave it at.